EDDI/ChangeLog.md

```diff
@@ -13,9 +17,9 @@
     * First installations will now take any custom VoiceAttack installation location into account when proposing a location for EDDI.
     * Upgrade installations will continue to use whatever location was selected in the first installation.
   * Speech Responder
-    * Added 'Jet cone damage' event
+    * Added `Jet cone damage` event
```

Comment:
We haven't in the past used code quotes for event names, but I'm not averse to it.